WcaOnRails/db/migrate/20220706232200_create_cached_results.rb

```diff
@@ -0,0 +1,12 @@
+# frozen_string_literal: true
+
+class CreateCachedResults < ActiveRecord::Migration[6.1]
+  def change
+    create_table(:cached_results, id: false) do |t|
+      t.string :key_params
+      t.json :payload
+      t.timestamps
+    end
+    execute "alter table cached_results add primary key (key_params);"
```

Comment:
No need for this, see my other comment above.

Reply:
Fixed